README.md

```diff
@@ -115,6 +123,16 @@ To see a step-by-step example, please see our example [here](./EXAMPLE.md)
 
 `./aperf report -h`
 
+**Custom-PMU Flags:**
+
+`-V, --version` version of Aperf
+
+`-p, --pmu-file` Name of the file for the custom PMU configuration
+
+`--verify` Verify the supplied PMU file
```

Comment:
maybe be a little more specific here: "Verify the supplied PMU file can be parsed.  Availability or functionality of the counters are not verified"